synapse/lib/ast.py

```diff
@@ -733,8 +733,9 @@ async def _compute(self, runt, path, limit):
                 retn.append(valunode.ndef[1])
 
                 if len(retn) > limit:
-                    mesg = f'Subquery used as a value yielded too many (>{limit}) nodes'
-                    raise self.addExcInfo(s_exc.BadTypeValu(mesg=mesg))
+                    query = self.kids[0].text
```

Comment:
should we limit the size of the query text here?

Reply:
I don't think we need to truncate the query text here. We're only adding the text from the subquery assignment. If it is truncated a user may have less data to search on in their own queries / codebase

Reply:
A subquery could be a 30 line nested construction expression. Lets do a quick goback.

Reply:
Per-discussion, cap this at 256 characters in the `mesg` parameter.